nmostesting/suites/IS0702Test.py

```diff
@@ -481,6 +485,222 @@ def test_05(self, test):
         else:
             return test.UNCLEAR("Not tested. No resources found.")
 
+    def test_06(self, test):
+        """MQTT senders on the same device have the same destination host/port
```

Comment:
Is this required by the IS-07 or IS-05 spec? It's certainly a good idea, but is there even a recommendation for this in the spec?

Reply:
no, was just copying the websocket tests

Reply:
Slightly different cases? I guess the assumption is that in all likelihood there _will_ be a single MQTT broker so single connections are the default, but that's a system issue as the Node is always an MQTT client. On the other hand, a Node with WebSocket Senders has to provide a WebSocket _Server_ and having a single `connection_uri` is what enables clients for Receivers to make a single connection, so worth testing?